apps/lockfile-explorer-web/src/containers/SelectedEntryPreview/styles.scss

```diff
@@ -1,11 +1,12 @@
 @import 'office-ui-fabric-core/dist/sass/References';
```

Comment:
In my web browser, the `Add Bookmark` button gets scrolled off the bottom of the pane.
![image](https://user-images.githubusercontent.com/4673363/202328313-96eb5943-ea30-4ee2-b7d6-6652e3ccb3b4.png)

Reply:
Moved the button to the top right of the panel